src/Microsoft.OData.Core/ODataMessageWriterSettings.cs

```diff
@@ -283,7 +283,7 @@ internal SelectedPropertiesNode SelectedProperties
             get
             {
                 return this.SelectExpandClause != null
-                    ? SelectedPropertiesNode.Create(this.SelectExpandClause)
+                    ? SelectedPropertiesNode.Create(this.SelectExpandClause, this.Version ?? ODataVersion.V4)

```

Comment:
?? maybe can't pass .net 3.5 compile.

Reply:
It is supported for 3.5 as well. At least it compiled just fine. 

---
In reply to: [256175616](https://github.com/OData/odata.net/pull/1402#discussion_r256175616) [](ancestors = 256175616)